src/components/__tests__/css-prop.test.js

```diff
@@ -0,0 +1,14 @@
+import styled from '@emotion/styled'
+import serializer from 'jest-emotion'
+import React from 'react'
+import { render } from 'react-testing-library'
+
+expect.addSnapshotSerializer(serializer)
+
+test('renders with correct styles', () => {
+  const H1 = styled.h1`
+    float: left;
+  `
+  const tree = render(<H1>hello world</H1>)
+  expect(tree).toMatchSnapshot()
```

Comment:
Please use `container.firstChild` instead for snapshot assertion, as shown in [react-testing-library's example](https://github.com/testing-library/react-testing-library#example)

```suggestion
  expect(container.firstChild).toMatchSnapshot()
```